pkg/controller/networkpolicy/antreanetworkpolicy.go

```diff
@@ -118,7 +118,9 @@ func (n *NetworkPolicyController) processAntreaNetworkPolicy(np *crdv1alpha1.Net
 		appliedToGroups = mergeAppliedToGroups(appliedToGroups, atgs...)
 		var peer *controlplane.NetworkPolicyPeer
 		if egressRule.ToServices != nil {
-			peer = n.svcRefToPeerForCRD(egressRule.ToServices, np.Namespace)
+			peer = n.svcRefToPeerForCRD(egressRule.ToServices, np.Namespace, crdv1alpha1.ScopeCluster)
+		} else if n.multiclusterEnabled && egressRule.ToMultiClusterServices != nil {
```

Comment:
As we discussed earlier, we need a new gate or config for stretched network policy, so this should be updated as well when the config is added.